src/amo/fonts/inter.scss

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-display: swap;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Inter Light'), url('woff2/Inter-Light.woff2') format('woff2'),
+    url('otf/Inter-Light.otf') format('opentype');
```

Comment:
Along the lines of @willdurand's question, do we still need both `woff2` and `otf` fonts?

Reply:
(going to aim to answer both questions here).  
So, right now we have otf, .woff, and .woff2 files (all multiple file, non-variable fonts).  A modern browser (Firefox 62+ Chrome 66+) supports all of those, plus a single* file variable font versions of woff2 - though to make things more complicated, variable fonts also have an OS dependency on MacOS (needs 10.13+).  To just support those modern browser+MacOS versions we only need variable woff2 fonts.

There's a decision around what older fonts we should bother supporting as fallback, if any:
 - If none then Firefox will just drops back to a generic sans-serif font on pre 62+macos10.13.
 - If one then we'd get the most coverage by choosing otf, (but it's huge!), or woff2 non-variable (Firefox 39+) for coverage of modern-ish browser/MacOS.
 - Or more than 1.  We have 4 types, so a selection or all 4.

* it's actually 2 files for Inter, because Italic variations are split out

Reply:
> If none then Firefox will just drops back to a generic sans-serif font on pre 62+macos10.13.

According to caniuse, the MacOS version requirement is a Firefox limitation: https://caniuse.com/?search=variable-font

> There's a decision around what older fonts we should bother supporting as fallback, if any:

We should target the browsers listed in the `.browerslist` file, which is: most recent and active browsers (last 2 versions) and _not_ IE. I see that variable fonts aren't supported on UC Browser and Opera Mini but I don't think it's a problem. The MacOS 10.13 requirement is a problem, though, because it is tied to Firefox and it's not a very old MacOS version.

If we keep the patch as it is now, we'd serve up to 2 fonts right? By default, modern browsers would use variable fonts with WOFF2, and the fallback would be OTF?

Reply:
the patch as it would offer 3 variants: `woff2` (variable), `woff2` (separate), `otf`.  Maybe we don't need 2 fallbacks though - if we drop `woff` (separate) then older Firefox/MacOS users would get served the much larger OTF font files; if we drop `otf` then super old browser users would see the default sans-serif font.

Reply:
ah ok, so maybe we don't need `woff`? Or we keep things are they are right now and see what we could do in the future? I guess that patch is still an improvement over what we have currently.

Reply:
yeah I think we'll leave this as the patch for now.  Having extra fallbacks won't hurt our performance (only older clients get them) so it's really just about tidiness and less cruft at that point - we can drop non-variable `woff2` or `otf` in a follow-up if we want.